Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,396 @@
+{
+    "name": "name of the copy of restorePointCollection resource",
+    "location": "location of the copy of the restorePointCollection resource",    
+    "tags": {
+        "department": "finance"
+    },
+    "properties": {
+         "source": {
+               "id": "/subscriptions/{subid}/resourceGroups/{resourceGroupName}/providers/microsoft.compute/restorePointCollections/{restorePointCollectionName}"
+         }
+    }
+}
+```
+
+#### Response
+The request response will include a status code and set of response headers.
+##### Status code
+The operation returns a 201 during create and 200 during Update.
+##### Response body
+```
+{
+    "name": "name of the copied restorePointCollection resource",
+    “id”: “CSM Id of copied restorePointCollection resource”,
+    "type": "Microsoft.Compute/restorePointCollections",
+    "location": "location of the copied restorePointCollection resource",    
+    "tags": {
+        "department": "finance"
+    },
+    "properties": {
+         "source": {
+               "id": "/subscriptions/{subid}/resourceGroups/{resourceGroupName}/providers/microsoft.compute/restorePointCollections/{restorePointCollectionName}"
+		"location" : “location of source RPC”
+         }
+    }
+}
+```
+
+### Create VM Restore Point in Target Region
+Next step is to trigger creation of a RestorePoint in the target RestorePointCollection referencing the RestorePoint in the source region that needs to be copied.
+
+#### URI request
+```
+PUT https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{restorePointCollectionName}/restorePoints/{restorePointName}&api-version={api-version}
+```
+#### Request body
+```
+{
+    "name": "name of the restore point resource",
+    "sourceRestorePoint": {
+               "id": "/subscriptions/{subid}/resourceGroups/{resourceGroupName}/providers/microsoft.compute/restorePointCollections/{restorePointCollectionName}/restorePoints/{restorePointName}"
+         }
+}
+```
+
+**NOTE:** Location of the sourceRestorePoint would be inferred from that of the source RestorePointCollection
+
+#### Response
+The request response will include a status code and set of response headers. 
+
+##### Status Code
+This is a long running operation; hence the operation returns a 201 during create. The client is expected to poll for the status using the operation. (Both the Location and Azure-AsyncOperation headers are provided for this purpose.) 
+
+During restore point creation, the ProvisioningState would appear as Creating in GET restore point API response. If creation fails, its ProvisioningState will be Failed. ProvisioningState would be set to Succeeded when the data copy across regions is initiated. 
+
+**NOTE:** The status of data movement can be tracked via “InstanceView” property. Restore Point is considered usable (can be used to restore a VM) only when copy of all the disk restore points are successful.
```

Comment:
"InstanceView" property doesn't clarify how to get that. Pls. add some more details or link to the GET InstanceView section.

"Restore Point is considered usable" --> Add VM to make it "VM Restore Point"

Reply:
Done